client/scripts/views/pages/profile/index.ts

```diff
@@ -245,15 +245,24 @@ const ProfilePage: m.Component<{ address: string, setIdentity?: boolean }, IProf
         if (chainInfo?.base === ChainBase.Substrate) {
           try {
             // TODO: should we enforce specific chain checksums here?
-            decodeAddress(address);
-            vnode.state.account = {
-              profile: null,
-              chain,
-              address,
-              id: null,
-              name: null,
-              user_id: null,
-            };
+            const decodedAddress = decodeAddress(address);
```

Comment:
You can remove the above TODO, now that we are enforcing specific chain checksums.

Reply:
done